doc/api/child_process.md

```diff
@@ -1602,6 +1606,7 @@ or [`child_process.fork()`][].
 [`process.env`]: process.html#process_process_env
 [`process.execPath`]: process.html#process_process_execpath
 [`process.send()`]: process.html#process_process_send_message_sendhandle_options_callback
+[Signal Events]: process.html#process_signal_events
```

Comment:
Once you've rebased, be sure to run `make lint-md` or `make test-doc` because I'm pretty sure the linter is going to say this line needs to be moved above all the others (because the are in ASCII order and `S` comes before ``` ` ```).